tests.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from environment_variables import TWITTER_CONSUMER_KEY, TWITTER_CONSUMER_SECRET, TWITTER_ACCESS_KEY, \
+    TWITTER_ACCESS_SECRET, FLICKR_API_KEY
+from main import flickr_get_photos_list, flickr_get_photo
+
+
+class TestEnvironmentVariables(unittest.TestCase):
```

Comment:
You don't really need these tests. os.getenv() always returns strings, and checking for non-empty values should be done in your app, not in tests.

Reply:
Changed